tests/foreman/api/test_syncplan.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=invalid-name
```

Comment:
this is added because the test names are too long.  I wanted to shorten the names but some folks are interested in keeping the long/descriptive names.  This is debatable.  

But for now, this workaround is to silence pylint.